Backend/src/main/java/com/login/demologin/controller/UserController.java

```diff
@@ -12,41 +14,45 @@
 
 @CrossOrigin(origins = "http://localhost:4200")
 @RestController
-@RequestMapping("/demo")
-public class MainController {
+@RequestMapping("/User")
```

Comment:
Usually this goes in plural (https://restfulapi.net/resource-naming/)

Reply:
thanks for check my project out richard ... I already fix your observations